pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -148,8 +149,12 @@ private Calendar getCalendar(ZoneOffset offset) {
       return calCache;
```

Comment:
of course with that change caching also breaks, so looks really like debuggig accidentally committed to this PR.

Reply:
yes, this is temporary. strangely enough all the DateTests passed on my local machine. I really wanted to see what else I broke

Reply:
I've reverted this